packages/web3-core-helpers/src/Formatters.js

```diff
@@ -302,8 +302,19 @@ export const outputBlockFormatter = (block) => {
     block.size = Utils.hexToNumber(block.size);
 
     // Support Quorum 2.2.0 - timestamp is not present in the Quorum getBlock response
-    if (block.timestamp !== null) {
+    if (block.timestamp) {
+      if (block.timestamp.length < 16) {
```

Comment:
You can merge line no. 305 with this line by making:-
`if(block.timestamp && block.timestamp.length < 16)`.

Also, why you are checking for `lock.timestamp.length < 16`?

Reply:
Using `block.timestamp` length to determine if the timestamp is nano based format. 
Made the amendments to check for length of 10. Also added additional test to increase test coverage.